bin/kalastatic

```diff
@@ -92,6 +94,51 @@ program
     })
   })
 
+function createFile(filename) {
+  fs.open(filename,'r',function(err, fd){
+    if (err) {
+      fs.writeFile(filename, '', function(err) {
+        if(err) {
+          console.log(err);
+        }
+        console.log("The file was saved!");
+      });
+    } else {
+      console.log("The file exists!");
+    }
+  });
+}
+
+function createComponent(dir,objectToCreate) {
+  var component = objectToCreate.split("/")
+  var basePath = path.join(dir, component[0], component[1])
+  mkdirp(basePath, function(){
+    var scssFile = path.join(basePath, "_"+component[1]+'.scss')
+    var jsonFile = path.join(basePath, component[1]+'.json')
+    var twigFile = path.join(basePath, component[1]+'.twig')
+    createFile(scssFile)
+    createFile(jsonFile)
+    createFile(twigFile)
```

Comment:
Looks like createFile() uses async methods. We'll either want to switch those over to the [writeFileSync](https://nodejs.org/docs/v0.3.1/api/fs.html#fs.writeFileSync) alternative, or handle the asynchronous callbacks.

Would also be neat to give it a base template. For example, give the scss file something like....

``` scss
/**
 * Template Styles
 *
 * This is where the styles go...
 *
 * Styleguide: MyStuff.STUFFS
 */

// Nothing
```
or something.